climpred/tests/test_stats.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import numpy as np
+import xarray as xr
+
+from climpred.stats import xr_rm_trend
+
+
+def test_xr_rm_trend_2d_dataarray():
+    da = xr.DataArray(

```

Comment:
for sample test data you can also use fixtures and parametrize https://docs.pytest.org/en/latest/parametrize.html (see test_prediction). you can basically just copy the inputs. da_ds and ds_ds are 4d.